lib/CL/devices/basic/basic.c

```diff
@@ -538,8 +538,13 @@ pocl_basic_notify (cl_device_id device, cl_event event, cl_event finished)
       return;
     }
 
-  if (!node->ready)
-    return;
+  if (1 != node->ready)
```

Comment:
```suggestion
  if (node->ready != 1)
```
Just a stylistic issue.